src/Microsoft.Fx.Portability/ObjectModel/DiagnosticAnalyzerInfo.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System.Collections.Generic;
+using System.Text.RegularExpressions;
+using System;
+
+namespace Microsoft.Fx.Portability.ObjectModel
+{
+#if FEATURE_SERIALIZABLE
+    [Serializable]
+#endif
+    public class DiagnosticAnalyzerInfo
+    {
+        public string AnalyzerName { get; set; }
+        public string Id { get; set; }
+        public ICollection<CompatibilityRangeAttribute> Attributes { get; set; }
+        public bool IsCompatibilityDiagnostic { get; set; }
```

Comment:
Can you add a ICollection<string> Languages? That way we can also track which languages an analyzer supports.